build.gradle.kts

```diff
@@ -3,13 +3,20 @@ plugins {
 }
 
 group = "com.github.leroyguillaume"
-version = "1.5.0"
+version = "1.5.1"
 
 repositories {
     mavenCentral()
 }
 
+configurations {
+    testImplementation.get().apply {
+        extendsFrom(configurations.compileOnly.get())
+    }
+}
+
 dependencies {
+    testImplementation("org.junit.jupiter:junit-jupiter-api:5.8.2")
```

Comment:
Can you move this line after the `Keycloak` block please?

Reply:
Done.